crates/ironrdp-graphics/src/pointer.rs

```diff
@@ -168,7 +168,11 @@ impl DecodedPointer {
             });
         }
 
-        if data.and_mask.len() != and_stride.length * usize::from(data.height) {
+        let default_and_mask = vec![0x00; and_stride.length * usize::from(data.height)];
```

Comment:
Question: is there was a bug?

Reply:
yeah, it crashes with ironrdp-server (mstsc works fine with it, without the AND mask)